### DISCUSSION TITLE
Unclear python (dev) API documentation

### DISCUSSION THREAD

**Discussion by Michelvl92** [NVlabs/instant-ngp#1242] [Q&A]
Hi,
It is unclear to where I can find the doc, and therefore not possible to build my own application based on pyngp. Als `scripts/run.py` does not provide much details and all the possibilities of the api. What is the best place to find this?

**Comment by AvishkarArjan** [comment]
Hey, same here. Wanted to explore a bit more about pyngp. Lemme know if you got to know anything
